Dockerfile

```diff
@@ -63,8 +63,8 @@ WORKDIR ${INSIGHTS_CODE_DIR}/
 # insights service config commands below
 RUN pip install  --no-cache-dir -r ${INSIGHTS_CODE_DIR}/requirements/production.txt 
 
-RUN nodeenv ${INSIGHTS_NODEENV_DIR} --node=18.20.2 --prebuilt \
-  && npm install -g npm@10.5.x
+RUN nodeenv ${INSIGHTS_NODEENV_DIR} --node=20.15.1 --prebuilt \
```

Comment:
As the Dockerfile has been moved to public-dockerfiles repo, node20 change would be needed there.

Reply:
This Node 20 update is required because the PR drops support for Node 18, and the Dockerfile is still active in this project. There’s a separate PR to remove the Dockerfile, it’s pending due to CI failures, for now Dockerfile is still part of this project and utilized in CI, aligning it with the Node version specified in the PR is necessary to ensure compatibility and prevent CI failures. Once that PR is merged and the Dockerfile is gone this won't matter, until then for as long as Dockerfile continues to exist, this change is relevant and essential.